app/src/main/res/menu/organiser_bottom_nav_menu.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/navigation_home"
+        android:icon="@drawable/nav_ic_calendar"
+        android:title="@null"/>
+
+    <item
+        android:id="@+id/navigation_dashboard"
```

Comment:
just wondering, why was the name dashboard chosen for add event and home for the calendar?

Reply:
this was from jens pr too, i guess I missed this when fixing things up